src/redux/reducers/main.js

```diff
@@ -12,7 +12,7 @@ import {
   UPDATE_FIELD,
 } from 'redux/constants';
 
-const defaultContextKey = 'tentDefault';
+const defaultContextKey = 'kinaseDefault';
```

Comment:
I wanted some way to make it clear that this was an internal name but can just call it `default` if you think that's better